examples/bri-3/src/bri/transactions/api/dtos/request/updateTransaction.dto.ts

```diff
@@ -0,0 +1,9 @@
+import { IsNotEmpty } from 'class-validator';
+
+export class UpdateTransactionDto {
```

Comment:
related to my comment above about what elements in a transaction could be changed @ognjenkurtic  

would require changes accordingly

Reply:
resolved